Sources/Core/UnsafePointer.swift

```diff
@@ -0,0 +1,13 @@
+// Copyright © 2021 Zoe Carver <zoe@zoecarver.com>.
+// All Rights Reserved.
+// SPDX-License-Identifier: BSD-3
+
+@frozen
+public struct UnsafePointer<Pointee> {
+  public let _rawValue: Builtin.RawPointer
```

Comment:
Actually, why does `_rawValue` have to be `public`?  Can we not get away with `private` or `internal`?

Reply:
FYI, I basically just copied and pasted these two classes from the "full" stdlib. They aren't public because we have a transparent constructor. I think it would be better to keep the constructor the same (and this the same) as the other stdlib, but I don't feel strongly, so I'm happy to remove the transparent attribute and make the member private.

Reply:
The signature should match - so, seems like the answer is that its `public` because the constructors are transparent?

Reply:
Yes, exactly.

Reply:
Thanks for the clarification!